components/Form.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function Form() {
+  const [title, setTitle] = useState('');
+  const [lyrics, setLyrics] = useState('');
+
+  async function handleSubmit(event) {
```

Comment:
do you need this to be an async function? I only saw them used while fetching data to use `await`.